contentprovider.go

```diff
@@ -658,34 +663,38 @@ func (p *contentProvider) calculateTermFrequency(cands []*candidateMatch, df ter
 	return termFreqs
 }
 
-func (p *contentProvider) candidateMatchScore(ms []*candidateMatch, language string, debug bool) (float64, string, []*Symbol) {
-	type debugScore struct {
-		what  string
-		score float64
-	}
+// scoredMatch holds the best-scoring candidate match, along with its score information.
```

Comment:
nit: scoredMatch seems more general than "just" the best-scoring match. Maybe we can drop the "best-scoring" from comment?

Reply:
Good point, updated this.